Documentation/aws.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Running Rook in AWS
+weight: 25
+indent: true
+---
+
+# Running Rook in AWS
+
+Rook can functionally replace Elastic Block Storage (EBS), Elastic File System (EFS) and S3 with Rook ObjectStore. Potentially reducing storage cost, increasing performance and portability of your Kubernetes workloads.
+
+:information_source:  
+Please refer to [Creating Rook Clusters][3] on how to customize your cluster to use specific devices.
+
+* To deliver robust performance at reasonable cost, consider using instances backed by [*instance store*][1] for Rook cluster nodes. Utilization of storage devices directly attached to the instance will reduce latency and provide more consistent IO performance.
+* The ephemeral nature of the *instance store* is handled by resiliency build into Rook. Make sure to configure `replicas` parameter in accordance with your durability requirements.
+* Multiple EBS devices can be [striped into RAID array][2] and used as Rook devices to improve performance of low cost EBS volumes.
+* Rook cluster nodes can be dedicated to manage your storage, or you can choose to utilize compute resources of Rook nodes to run Kubernetes workloads with proper resource isolation, father reducing your cluster costs.
+
+## Benefits of using Rook instead of EC2 storage services (EBS, EFS, S3)
+
```

Comment:
Is the purpose of this doc to guide users to install and use rook in aws? I would expect instructions or links to instructions on how to install kubernetes on aws and how to install rook. Also what kind of images, storage type, flavors of aws resources work best with rook.

Reply:
Rook project is not in a business of directing of how to install or run Kubernetes. As for Rook, the installation is the same for all platforms, there are no provider specific instructions.

Reply:
I understand that rook project is not in the business or installing kubernetes and it should behave the same no matter the platform but since this document is specifically about AWS, i feel it should have a link directing users on how to install it.. Possibly a link to https://github.com/kubernetes-incubator/kube-aws or whatever is the most popular tool to deploy k8s on aws.  Kinda similar to what we have here https://github.com/rook/rook/blob/master/Documentation/k8s-pre-reqs.md